### DISCUSSION TITLE
Regarding student developer pack benefits

### DISCUSSION THREAD

**Discussion by shweta0dixit** [community/community#134845] [GitHub Education]
### Select Topic Area

Question

### Body

I applied for the GitHub Student Developer Pack on August 2, and my status was approved. However, I haven't received access to the benefits yet. Could someone guide me on what to do next or how long it typically takes to receive access after approval ,already 4 days has been passed.

Thank you!

**Comment by Irissuu** [comment]
Check your email for any updates, Go to the [GitHub Education page](https://education.github.com/pack) and try logging in there to see if you can access the benefits, If you still haven't received any instructions,  reach out to GitHub Support. You can contact them through their [support page](https://support.github.com/)

**Comment by ebndev** [comment]
Hi @shweta0dixit, thanks for posting in GitHub Discussions! 

I've gone ahead and moved this post to our `GitHub Education` category, as this topic might be more relevant there and might help someone give you a nudge in the right direction 😁
